website_sale_product_detail_attribute_image/views/product_attribute_views.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="attribute_tree_view" model="ir.ui.view">
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.attribute_tree_view"/>
+        <field name="arch" type="xml">
+            <field name="create_variant" position="after">
+                <field name="website_product_detail_image"/>
```

Comment:
Why not uses widget='image' here? size?

Reply:
is it working on trees?

Reply:
Work but without resize image :/ perhaps can use: https://github.com/OCA/web/tree/11.0/web_tree_image

Reply:
Well, maybe too much overhead for this, don't you think? Let Sergio decides.

Reply:
I don't want dependencies of other modules, If the user wants to view the image only has to do click on record, also this model, once it has been configured the user not will coming back to see the images every day.
But I thought so too :smile: